filecoin-proofs/src/api/internal.rs

```diff
@@ -238,34 +241,39 @@ pub fn public_params(
     ZigZagDrgPoRep::<DefaultTreeHasher>::setup(&setup_params(sector_bytes)).unwrap()
 }
 
-type PostSetupParams = vdf_post::SetupParams<PedersenDomain, vdf_sloth::Sloth>;
-pub type PostPublicParams = vdf_post::PublicParams<PedersenDomain, vdf_sloth::Sloth>;
+type PostSetupParams = beacon_post::SetupParams<PedersenDomain, vdf_sloth::Sloth>;
+pub type PostPublicParams = beacon_post::PublicParams<PedersenDomain, vdf_sloth::Sloth>;
 
 const POST_CHALLENGE_COUNT: usize = 30;
-const POST_EPOCHS: usize = 3;
+const POST_EPOCHS: usize = 0;
+const POST_PERIODS_COUNT: usize = 1;
```

Comment:
@nicola need a number here

Reply:
@dignifiedquire @nicola - What's the relationship between `POST_PERIODS_COUNT` and total time to generate a PoSt? I ask this question because PoSt generation takes very little time right now. go-filecoin relies on that fact. If PoSt generation starts to take approximately the whole proving period, we're going to need to change a bunch of software.